packages/neos-ui/src/Containers/LeftSideBar/NodeTreeSearchBar/NodeTreeSearchInput/style.css

```diff
@@ -30,3 +24,16 @@
     left: var(--spacing-Full);
     height: 40px;
 }
+
+.wrapper {
+    display: flex;
+    border-right: 1px solid var(--colors-ContrastDark);
+    position: relative;
+    width: 153px;
```

Comment:
Personally I'm not super happy about magic numbers like these. It's always better to use percentages or at least calc + constants, but I guess we can improve the code quality later, not to let this PR rot.

Reply:
Hmm our goldenUnit is 40px... maybe we can use this as base. The padding before and the 160px can also use the goldenUnit with calc.